models/exchange/coinbase_pro/api.py

```diff
@@ -355,7 +355,8 @@ def marketBaseIncrement(self, market, amount) -> float:
         else:
             nb_digits = 0
 
-        return float(f'%.{nb_digits}f'%(amount))
+        from math import floor
```

Comment:
This import  should be at the beginning of the file with all other imports.

Reply:
Gonna be honest, never actually coded in Python but loving the project and wanted to help.
Be sure to point out any other errors I may do along the way!